packages/world/src/ProgramInterfaces.sol

```diff
@@ -35,10 +36,8 @@ interface ITransferHook {
   function onTransfer(
     EntityId caller,
     EntityId target,
-    EntityId from,
-    EntityId to,
-    ObjectAmount[] memory objectAmounts,
-    EntityId[] memory entities,
+    SlotData[] memory deposits,
```

Comment:
What does this look like for a transfer within the same chest? ie moving items around

As a chest program writer, is that a "depsoit"/"withdraw"? or I guess both would be filled with the same items?

Reply:
if you move stuff within the chest's inventory, let's say the object in slot 0 to slot 1, you would see deposits[0] == withdrawals[0].